backend/routes/api/resources.js

```diff
@@ -87,6 +88,12 @@ router.post(
   }),
   errorWrap(async (req, res) => {
     const data = req.body;
+    data.tags = extractor.extract(data.notes, {
+      language: "english",
+      remove_digits: true,
+      return_changed_case: true,
+      remove_duplicates: true
+    });
     const newResource = new Resource(data);
```

Comment:
Very small nit for best style: rather than putting a new field in the `req.body`, it's best if we instead use the spread syntax and create a new variable to hold `tags`. Then we can do `const newResource = new Resource({...data, tags})`. This way, you're copying the body of the request rather than mutating it. The only time this would matter is like if another function were to be using `req.body` but here it's totally chill—just something to be aware of.